src/main/java/seedu/command/EventCommandInterpreter.java

```diff
@@ -28,15 +37,35 @@ public Command decideCommand(String commandDescription) throws DukeException {
             event = eventParser.parseEvent(commandParameters);
             command = new AddEvent(event, this.eventList);
             break;
-        case "edit":
-            //TODO edit
+        case "editname":
+            index = eventParser.parseIndex(commandParameters);
+            name = eventParser.parseEventName(commandParameters);
+            command = new EditName(index, name, this.eventList);
+            break;
+        case "editdatetime":
+            index = eventParser.parseIndex(commandParameters);
+            datetime = eventParser.parseEventDateTime(commandParameters);
+            command = new EditDateTime(index, datetime, this.eventList);
+            break;
+        case "editvenue":
+            index = eventParser.parseIndex(commandParameters);
+            venue = eventParser.parseVenue(commandParameters);
+            command = new EditVenue(index, venue, this.eventList);
+            break;
+        case "editevent":
+            index = eventParser.parseIndex(commandParameters);
+            event = eventParser.parseEvent(commandParameters);
+            command = new EditEvent(index, event, this.eventList);
```

Comment:
are we keeping all commands in lowercase, or camelCase? We haven't really discussed about this.

Reply:
it dosent matter how the user inputs the commands, its all turned into lower case and compared against the diff command cases